package-lock.json

```diff
@@ -1020,7 +1020,8 @@
     "core-util-is": {
```

Comment:
just a general question: did you make changes to this file manually, or was this the result of running `npm <command>`? if the latter, which command did you use?

Reply:
`npm rm pipeline-builder`, I'd wager

Reply:
(lockfiles should never be modified manually)

Reply:
npm rm pipeline-builder